day6/index.js

```diff
@@ -0,0 +1,45 @@
+$(() => {
+  const dict = ['apprehensive', 'barricade', 'condition', 'dictionary', 'fragile', 'grueling', 'habitation', 'illuminate',
+                'jostle', 'kindle', 'luminous', 'materialize', 'outlandish', 'plenteous', 'question', 'restaurant',
+                'seperate', 'turtle','umbrella','versatile','weather','xray','yoyo','zepalz']
+
+  const randomWord = () => {
+    let random = dict[Math.floor((Math.random() * (dict.length)))]
+    for (var i = 7; i > 0; i--) {
+      $('#used').append(`<span id="used${i}">_ </span>`)
+    }
+    for (var i = 0; i < random.length; i++) {
+      if (random.includes(random[i]))
+        $('#answer').append(`<span class="ans${random.indexOf(random[i])}">_ </span>`)
+    }
+    return random;
+  }
+
+  const word = randomWord();
+  let used = ""
+
+  $(document).on('keydown', () => {
+    var key = event.key
+    var left = $('#left').html() - 1
+
+    if (event.keyCode >= 65 && event.keyCode <= 90) {
+      if (word.includes(key)) {
+        $(`.ans${word.indexOf(key)}`).html(key + ' ')
+      } else if (!used.includes(key)) {
+        used += key
```

Comment:
This approach gives a bug when trying to press special commands like `Ctrl+Alt+i`. Try it and you will see the word `Dead` (like your papa) appear on the screen. Let's dig in to what causing it.

When pressing `Ctrl+Alt+i`, the `event.keyCode` is `73` which is associated to letter `i`, which is correct. But with the combination keys (`ctrl`, `alt`, etc.), it gives a different value to `event.key` attribute, instead of having value `i`, it gives you value of `Dead` (still, like your papa).

So as we know, the source of truth is the `event.keyCode`, not `event.key`, we should find a way to convert from number to associated character. Luckily, you could use Javascript's `String.fromCharCode()`.
Ex:
```
// event.keyCode => 73
String.fromCharCode(event.keyCode) // "i"
```